todo_list/views/views_main.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
```

Comment:
@quratulain25 Don't rename it `views_main.py` since this file is already under `views/` directory/package so no need to add a prefix.